oteapi/models/triplestoreconfig.py

```diff
@@ -1,13 +1,13 @@
 """Pydantic TripleStore Configuration Data Model."""
-from typing import TYPE_CHECKING
+from typing import Annotated, Optional
 
-from pydantic import Field, root_validator
+from pydantic import Field, model_validator
 
 from oteapi.models.genericconfig import GenericConfig
-from oteapi.models.secretconfig import SecretConfig
+from oteapi.models.secretconfig import SecretConfig, TogglableSecretStr
 
-if TYPE_CHECKING:  # pragma: no cover
-    from typing import Any, Dict
+ExcludeTogglableSecretStr = Annotated[Optional[TogglableSecretStr], Field(exclude=True)]
+"""Annotated type alias for excluding a togglable secret from serialization."""
```

Comment:
```
# Defines a type alias that excludes a togglable secret from being serialized.
ExcludeTogglableSecretStr = Annotated[Optional[TogglableSecretStr], Field(exclude=True)]

```

Reply:
The way it's done now, means the type alias will have it's own doc-string in the documentation:
![image](https://github.com/EMMC-ASBL/oteapi-core/assets/43357585/fce1466c-6eeb-4adf-aa71-001847647612)